lib/features/app/di/app_scope.dart

```diff
@@ -63,11 +63,11 @@ class AppScope implements IAppScope {
     _errorHandler = DefaultErrorHandler();
     _router = AppRouter.instance();
     _themeModeStorage = ThemeModeStorageImpl(_sharedPreferences);
-    _analyticsService = AnalyticsServiceImpl([
+    _analyticsService = AnalyticService.withStrategies({
       // TODO(init): can be removed MockFirebaseAnalytics and MockAmplitudeAnalytics, added for demo analytics track
-      FirebaseAnalyticTracker(MockFirebaseAnalytics()),
-      AmplitudeAnalyticTracker(MockAmplitudeAnalytics()),
-    ]);
+      FirebaseAnalyticStrategy(MockFirebaseAnalytics()),
+      AmplitudeAnalyticStrategy(MockAmplitudeAnalytics()),
```

Comment:
Let's remove Amplitude analytics from template, I think it is redundant, one example with Firebase it enough.